### ISSUE TITLE
В поле order_entity.payment_id записывается значение поля credit_request_entity.bank_id при оплате заявки в кредит.

### ISSUE THREAD

**Issue by FlyingDog265** [FlyingDog265/NetologyDiploma#4]
В поле _order_entity.payment_id_ записывается значение поля _credit_request_entity.bank_id_ при оплате заявки в кредит.

**Шаги:**
1. В браузере перейти по адресу localhost:8080
1. Кликнуть "Купить в кредит"
1. Оформить заявку используя карту с валидными данными:
    * В поле "Номер карты" ввести: 4444 4444 4444 4441
    * В поле "Месяц" ввести: 08
    * В поле "Год" ввести: 22
    * Поле "Владелец" ввести: Ivanov Ivan
    * В поле "CVC/CVV" ввести: 999
    * Нажать кнопку "Продолжить"
1. Дождаться сообщения об успешной операции
1. Перейти в БД

**Ожидаемый результат**
* В таблице _order_entity_ содержится запись о транзакции
* _order_entity.payment_id_ значение идентичное _credit_request_entity.id_ 

**Фактический результат**
* В таблице _order_entity_ содержится запись о транзакции
* _order_entity.payment_id_ значение идентичное _credit_request_entity.bank_id_ 

**Окружение**
* ОS Windows 10 version 2020 64-bit
* Browser Google Chrome Версия 91.0.4472.164
* openjdk version "11.0.9" 2020-10-20

**Скрины**
![5](https://user-images.githubusercontent.com/44607099/126769393-a12cda34-521e-4122-8add-7ba9b0eebacd.jpg)

**Comment by ArRomanov** [comment]
Да, там действительно есть баг. У нас три таблицы, две (для дебетовых и кредитных транзакций) ссылаются на одну с общими транзакциями, но в нашем случае приложение с обеих форм пишет id транзакции в одно и тоже поле общей таблицы.
